frontend/src/concepts/hardwareProfiles/HardwareProfileSelect.tsx

```diff
@@ -160,39 +145,375 @@ const HardwareProfileSelect: React.FC<HardwareProfileSelectProps> = ({
+                        maxLines={2}
+                        content={
+                          hardwareProfileConfig.selectedProfile.spec.description ||
+                          (hardwareProfileConfig.selectedProfile.spec.identifiers &&
+                            hardwareProfileConfig.selectedProfile.spec.identifiers
+                              .map((identifier) =>
+                                formatResource(
+                                  identifier.displayName,
+                                  identifier.defaultCount.toString(),
+                                  identifier.defaultCount.toString(),
+                                ),
+                              )
+                              .join('; '))
+                        }
+                      />
+                    </HelperTextItem>
+                  </HelperText>
+                </FormHelperText>
+              ) : hardwareProfileConfig.useExistingSettings ? (
+                'Use existing resource requests/limits, tolerations, and node selectors.'
+              ) : null}
+              {(hardwareProfilesError || currentProjectHardwareProfilesError) && (
+                <HelperText isLiveRegion>
+                  <HelperTextItem variant="error">Error loading hardware profiles</HelperTextItem>
+                </HelperText>
+              )}
+            </>
+          ) : (
+            <>
+              <SimpleSelect
+                dataTestId="hardware-profile-select"
+                previewDescription={previewDescription}
+                options={options}
+                value={
+                  hardwareProfileConfig.selectedProfile?.metadata.name ??
+                  (hardwareProfileConfig.useExistingSettings ? EXISTING_SETTINGS_KEY : undefined)
+                }
+                onChange={(key) => {
+                  if (key === EXISTING_SETTINGS_KEY) {
+                    onChange(undefined);
+                  } else {
+                    const profile = hardwareProfiles.find((hp) => hp.metadata.name === key);
+                    if (profile) {
+                      onChange(profile);
+                    }
+                  }
+                }}
+                placeholder={
+                  options.length > 0
+                    ? 'Select hardware profile...'
+                    : hardwareProfilesError
+                    ? 'Error loading hardware profiles'
+                    : 'No enabled or valid hardware profiles are available. Contact your administrator.'
+                }
+                isFullWidth
+                isSkeleton={!hardwareProfilesLoaded && !hardwareProfilesError}
+                isScrollable
+              />
+              {hardwareProfilesError && (
+                <HelperText isLiveRegion>
+                  <HelperTextItem variant="error">Error loading hardware profiles</HelperTextItem>
+                </HelperText>
+              )}
+            </>
+          )}
```

Comment:
this tsx gets a bit big and could maybe be factored out a bit... but okay with future story to do so